R/ff_prep_refactored.R

```diff
@@ -80,11 +80,10 @@ ff_prep_refactored <- function(datafolder = NA, country = NA, shape = NA, tiles
 
   shape <- result$shape
   tiles <- result$tiles
-  tilesvect <- result$tilesvect
 
   ########## list files and exclude features######
 
-  allfiles <- list_files_and_exclude_features(datafolder = datafolder, tiles, groundtruth_pattern, verbose)
+  allfiles <- list_and_filter_tile_files(datafolder = datafolder, tiles, groundtruth_pattern, verbose)
```

Comment:
It seems to not necessary list and filter files. To me it seems like it just gathers the input and ground_truth tiles

Reply:
there are two comments inside the function, first, they list the input and ground truth and then filter it with a pattern, what naming do you suggest?

Reply:
Those comments do say that yeah, but it does sth else if you look at the code. It kind of " collects"  the files. So maybe sth like (collect_data_and_groundtruth_files)? Not perfect either.

Reply:
list and filter sounds more natural to me, but I'll change if you insist

Reply:
This function lists all feature files (file names) and the files of the selected ground truth pattern for the specified tiles. So, it only filters out the not wanted ground truth files.  Maybe just list_files would be a good name?